<?xml version="1.0"?>
<launch>
    <!-- 参数设置 测试用 /dev/pts/5 (虚拟串口) 实机用 /dev/ttyUSB0 (实际串口) -->
    <!-- 串口通信节点 -->
    <node pkg="serial_com" type="serial_com" name="serial_com" output="screen">
        <!-- 参数设置 -->
        <param name="cmd_vel_topic" type="string" value="/cmd_vel1" />
        <param name="serial_port" type="string" value="/dev/pts/4" />
        
        <!-- ROS参数服务器设置 -->
        <rosparam>
            baud_rate: 115200
            timeout: 1000
            default_spin_mode: false
        </rosparam>
    </node>
    
</launch>